__tests__/app.test.js

```diff
@@ -43,4 +43,27 @@ describe('/api', () => {
                 expect(response.body.apiData).toEqual(apiEndpointsJSON)
             })
     });
+});
+
+describe('/api/articles/:article_id', () => {
+    test('GET: 200 an article object with all associated properties', () => {
+        return request(app)
+        .get('/api/articles/1')
+        .expect(200)
+        .then((response) => {
+            const {rows} = response.body
+            rows.forEach(row => {
```

Comment:
We want to explicitly check here we are getting the correct article back. We can do this by checking the article id is 1